### DISCUSSION TITLE
I can't make release with files (assets)

### DISCUSSION THREAD

**Discussion by kozicpetar** [community/community#25995] [Actions]
<p>Hi folks,<br>
I have one problem with creating release. I will try to describe my workflow.<br>
I have workflow where I download some files from net and I build some libraries. When I built all I get two libraries files.<br>
That built libraries files I storing in ./build/ folder in repository.<br>
That files I need to put as new release on github repository.</p>
<p>I finish all jobs but when I try to create new release with:</p>
<pre><code class="lang-auto">- name: Create Release
    id: create_release
    uses: actions/create-release@v1
    env:
      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
    with:
      tag_name: ${{ steps.version.outputs.some_my_variables }}-${{ steps.version.outputs.some_my_variables }}
      release_name: ${{ steps.version.outputs.some_my_variables }}-${{ steps.version.outputs.some_my_variables }}
      body: |
        My body message
      draft: false
      prerelease: false
</code></pre>
<p>release was created but in .zip and .tar.gz I only can see actuall repostiory files. I need to have only that my two libraries files from ./build/ folder.<br>
How can I do that?</p>
<p>P.S. I tried with some actions@ for upload files in release but that is not acceptable to me.</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Releases are based on <a href="https://git-scm.com/book/en/Git-Basics-Tagging" rel="nofollow noopener">Git tags</a>, which mark a specific point in your repository’s history. In my opinion, artifacts is more suitable for your scenario.  You could use <a href="https://github.com/actions/upload-artifact" rel="nofollow noopener">upload artifacts action</a> with the specific path.<br>
You could download the artifacts from the workflow run result page:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/9/9f82756d1586c5a08207f901f6a891b9e21dd86a.png" data-download-href="/uploads/short-url/mL5qiSVQR2qCS0qlXcVPo0Ehhb4.png?dl=1" title="artifacts" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24806478/181094454-ab312731-4c12-4c43-a8b0-f8a37f8d02f0.png" alt="artifacts" data-base62-sha1="mL5qiSVQR2qCS0qlXcVPo0Ehhb4" width="690" height="148" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/9/9f82756d1586c5a08207f901f6a891b9e21dd86a_2_690x148.png, https://aws1.discourse-cdn.com/github/optimized/2X/9/9f82756d1586c5a08207f901f6a891b9e21dd86a_2_1035x222.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/9/9f82756d1586c5a08207f901f6a891b9e21dd86a_2_1380x296.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/9/9f82756d1586c5a08207f901f6a891b9e21dd86a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">artifacts</span><span class="informations">1904×409 18.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by kozicpetar** [comment]
<p>Sorry on delay.<br>
Thank you on answer, but this is not acceptable by me.<br>
In anyway thank you.</p>
